requirements.txt

```diff
@@ -6,3 +6,4 @@ pytest-cov==2.6.1
 capturer==2.4
```

Comment:
General notes before merging:
- Check this on auto-instrument (in node we had issues over there)

Reply:
will do! Thanks :)